app/views/users/backup_code_setup/create.html.erb

```diff
@@ -14,21 +14,19 @@
 </p>
 
 <div class="margin-top-4 margin-bottom-5">
-  <table>
-    <tr>
-      <% [ @codes.first(@codes.length / 2), @codes.last(@codes.length / 2)].each do |section| %>
-        <td class="padding-0">
-          <% section.each do |code| %>
-            <code class="bold h2">
-              <%= RandomPhrase.format(code, separator: '-') %>
-            </code>
-            <br/>
-            <br/>
-          <% end %>
-        </td>
-      <% end %>
-    </tr>
-  </table>
+  <div class="grid-row">
+    <% [ @codes.first(@codes.length / 2), @codes.last(@codes.length / 2)].each do |section| %>
+      <div class="grid-col-6">
```

Comment:
How does this look on mobile? Should it be collapsing to single column?

```suggestion
      <div class="grid-col-12 tablet:grid-col-6">
```

Reply:
I coded it to match what we currently have on prod, which is 2 column on mobile
![Screen Shot 2022-02-16 at 12 35 03 PM](https://user-images.githubusercontent.com/7727136/154326488-adfe7dca-9e8e-4704-8286-8bbd6d5f0da0.png)
:

Reply:
I see. Fair enough. Might be something to look at separately, since it gets kinda tight at small phone sizes like iPhone SE. FYSA @anniehirshman-gsa 

![image](https://user-images.githubusercontent.com/1779930/154332622-85479c27-80f5-4f4e-863c-705635266d73.png)

Reply:
Per our discussion with @anniehirshman-gsa we decided to make this change in this pull request and update the ticket.